src/test/java/api/support/fakes/FakeStorageModule.java

```diff
@@ -95,7 +95,7 @@ void register(Router router) {
     router.get(rootPath).handler(this::checkForUnexpectedQueryParameters);
     router.get(rootPath).handler(this::getMany);
 
-    router.delete(rootPath).handler(this::empty);
+    router.delete(rootPath).handler(this::deleteMany);
```

Comment:
Given that only one API implements this feature at the moment, I think this could be conditional only for some fake APIs

Reply:
As "empty" and "deleteMany" has the same path and method, do you think we should register one of them based on some kind of flag?

Reply:
Yes please, there is a builder used for configuring fake modules.

Reply:
Done